src/ApparentHorizons/StrahlkorperInDifferentFrameHelpers.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+template <size_t Dim>
+class Domain;
+namespace domain::FunctionsOfTime {
+class FunctionOfTime;
+}  // namespace domain::FunctionsOfTime
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace strahlkorper_in_different_frame_helpers {
+template <typename SrcFrame, typename DestFrame>
+void coords_to_different_frame(
```

Comment:
Can you add a small amount of docs? Just saying that this loops over the blocks of the domain and transforms the coords from the source frame to the destination frame.